Index: venture/python/lib/engine/engine.py
===================================================================
--- venture.orig/python/lib/engine/engine.py	2015-07-14 18:16:20.029334741 +0000
+++ venture/python/lib/engine/engine.py	2015-07-14 18:16:26.145334975 +0000
@@ -15,7 +15,7 @@
 # You should have received a copy of the GNU General Public License
 # along with Venture.  If not, see <http://www.gnu.org/licenses/>.
 import threading
-import dill
+import cPickle as dill
 import time
 from contextlib import contextmanager
 
